packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { TWalletInfo } from 'Types';
+import { useCurrencyConfig } from '@deriv/api';
```

Comment:
replace `useCurrencyConfig` from `@deriv/hooks` to `useCurrencyConfig` from `@deriv/api`